gateway-service-knoxtoken/src/main/java/org/apache/knox/gateway/service/knoxtoken/TokenResource.java

```diff
@@ -228,6 +235,9 @@ public void init() throws AliasServiceException, ServiceLifecycleException, KeyL
       }
     }
 
+    String shouldIncludeGroupsParam = context.getInitParameter(TOKEN_INCLUDE_GROUPS_IN_JWT);
+    shouldIncludeGroups = shouldIncludeGroupsParam == null ? false : Boolean.parseBoolean(shouldIncludeGroupsParam);
```

Comment:
It looks like this is defaulting to false, and the explicit false in the homepage topology should be unnecessary.

Reply:
We can remove it from there. It only serves as "documentation" of a new config parameter.